### PR TITLE
Added few js code snippets

### PR DESCRIPTION
**Pull request by akhilender-bongirwar**

### CODE REVIEW

File: JS/andOr.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+let isFun = true;
+let isLearning = 0;
+
+console.log(isFun && "Learning is fun!");
+console.log(isFun || "Learning is not fun!");
```
Comment (rootCircle) [inline JS/andOr.js:5]: Did you meant to use isLearning here, instead of isFun? (isLearning variable is not used in the entire code, so was doubtful!)

**Review by rootCircle — CHANGES_REQUESTED** [review]
Check in mentioned changes before merging!

File: JS/andOr.js (hunk 1 above)
Comment (akhilender-bongirwar) [reply JS/andOr.js:5]: Yeah changing it : )..

**Review by rootCircle — APPROVED** [review]

**Comment by rootCircle** [comment]
LGTM :-) Merged
Thanks for your contribution!

**Pull request merged**